samples/led/src/main.c

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <zephyr/logging/log.h>
+#include <zephyr/kernel.h>
+#include <npmx_driver.h>
+#include <npmx_led.h>
+
+#define LOG_MODULE_NAME pmic_led
+LOG_MODULE_REGISTER(LOG_MODULE_NAME);
+
+/**
+ * @brief Set LEDs status based on lower 3 bits of status variable.
+ *
+ * @param[in] p_pm   Pointer to the instance of PMIC device.
+ * @param[in] status Lower 3 bits are used to set or reset LEDs.
+ */
+void leds_status_set(npmx_instance_t *p_pm, uint8_t status)
+{
+	npmx_led_state_set(npmx_led_get(p_pm, 0), (status & (1UL << 0UL)) > 0);
+	npmx_led_state_set(npmx_led_get(p_pm, 1), (status & (1UL << 1UL)) > 0);
+	npmx_led_state_set(npmx_led_get(p_pm, 2), (status & (1UL << 2UL)) > 0);
+}
+
+void main(void)
+{
+	const struct device *pmic_dev = DEVICE_DT_GET(DT_NODELABEL(npm_0));
+
+	if (!device_is_ready(pmic_dev)) {
+		LOG_INF("PMIC device is not ready.");
+		return;
+	}
+
+	LOG_INF("PMIC device OK.");
+
+	/* Get the pointer to npmx device. */
+	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
+
+	/* Leds sequence to set. Only first 3 bits are used. */
```

Comment:
```suggestion
	/* Set the LEDs sequence. Only first 3 bits are used. */
```
To be confirmed if ok.